config.toml

```diff
@@ -33,9 +33,32 @@
    # also (minimum) hysteresis nodes for the minimum number of nodes
    SwitchHysteresisForMinNodesEnableEpoch = 1
 
+   # TransactionSignedWithTxHashEnableEpoch represents the epoch when the node will also accept transactions that are
+   # signed with the hash of transaction
+   TransactionSignedWithTxHashEnableEpoch = 23
+
+   # MetaProtectionEnableEpoch represents the epoch when the transactions to the metachain are checked to have enough gas
+   MetaProtectionEnableEpoch = 23
+
+   # AheadOfTimeGasUsageEnableEpoch represents the epoch when the cost of smart contract prepare changes from compiler per byte to ahead of time prepare per byte
+   AheadOfTimeGasUsageEnableEpoch = 23
+
+   # GasPriceModifierEnableEpoch represents the epoch when the gas price modifier in fee computation is enabled
+   GasPriceModifierEnableEpoch = 23
+
+   # TO BE CHANGED IN MAINNET AND PUBLIC TESTNET CONFIGS
+   # MaxNodesChangeEnableEpoch holds configuration for changing the maximum number of nodes and the enabling epoch
+   MaxNodesChangeEnableEpoch = [
+        { EpochEnable = 0, MaxNumNodes = 437, NodesToShufflePerShard = 23 },
+        { EpochEnable = 23, MaxNumNodes = 450, NodesToShufflePerShard = 23 }
```

Comment:
112 nodes per shard and ~90 eligible and 23 max shuffled in the waiting list?

Reply:
yup